### DISCUSSION TITLE
Visual Studio sufferers -> install the "Restart Visual Studio" Extension

### DISCUSSION THREAD

**Discussion by cmp-nct** [community/community#39438] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Anyone using Copilot on Visual Studio likely suffers through hundreds of hangs when the plugin becomes unstable after half an hour of use.
Install the Restart Extension, it allows to restart Visual Studio from the "File" menu and it will put your caret into the same position and window as it was before.
Saves a lot of nerves given you need to restart the IDE so often to use it with Copilot, it's still very annoying due to the loading delay but you can get a cup of coffee or similar and get back to work where you (copilot) stopped.

**Comment by iamcjwilliams** [comment]
@cmp-nct Thank you for your feedback, Our copilot team is working to improve the copilot VS experience

**Reply by cmp-nct** [reply]
Some of the most pressing issues are literally a 30 minute exercise and they have not been done in a year. 
The plugin changes mostly involve authentication, so someone seems to invest a bit of time into them but the actual problems are all ignored.
VS is especially unstable but the problems are with all plugins.
